CMakeLists.txt

```diff
@@ -36,6 +36,17 @@ if (MSVC)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -wd4125")
 endif()
 
+# Define preprocessor for accelerators.
+set(NNPA_ENABLED 0)
```

Comment:
I commented later in the code to init the passes, but the same applies here. We shouldn't be polluting the top level of onnx-mlir with knowledge about the accelerators.

Reply:
Agree. I think we can make the accelerator code functionally work in this PR, and then address this point in a different PR. I am saying so because I think landing this one will allow us to make progress in parallel later.